package.json

```diff
@@ -21,7 +21,6 @@
     "design-tokens:build": "yarn --cwd design-system build:tokens",
     "design-tokens:build:watch": "yarn --cwd design-system build:tokens:watch",
     "prebuild": "yarn generate-icons && yarn design-tokens:build && yarn i18n:compile && yarn generate-readmes",
-    "prestorybook:start": "yarn prebuild",
```

Comment:
As suggested by @jonnybel.
Didn't want to open a separate PR for this.

Reply:
Thanks!